docs/manual/src/proc_macro/index.md

```diff
@@ -91,13 +91,43 @@ trait MyTrait {
 
 ```
 
-Most UniFFI [builtin types](../udl/builtin_types.md) can be used as parameter and return types.
-When a type is not supported, you will get a clear compiler error about it.
+All UniFFI types can be used as parameter and return types.
```

Comment:
What are `UniFFI types`? ~~Also we don't support returning references, but we can take them in.~~ (this is made more clear in the next sentence)
So probably this sentence should be more precise.

Reply:
Good point, I just pushed an update.